src/mgr/BaseMgrModule.cc

```diff
@@ -401,7 +401,7 @@ ceph_get_module_option(BaseMgrModule *self, PyObject *args)
   char *module = nullptr;
   char *key = nullptr;
   char *prefix = nullptr;
-  if (!PyArg_ParseTuple(args, "sss:ceph_get_module_option", &module, &key,
+  if (!PyArg_ParseTuple(args, "ss|s:ceph_get_module_option", &module, &key,
```

Comment:
```
"z" (string or None) [char *]
Like "s", but the Python object may also be None, in which case the C pointer is set to NULL.
```
from https://docs.python.org/2.0/ext/parseTuple.html

Reply:
maybe it doesn't matter, just saw this while looking into http://tracker.ceph.com/issues/38705

Reply:
We already use ``z`` in other methods, so i changed it.